rom/dev/tests/smoke_test.rs

```diff
@@ -0,0 +1,24 @@
+// Licensed under the Apache-2.0 license
+
+use caliptra_builder::{ImageOptions, APP_WITH_UART, FMC_WITH_UART, ROM_WITH_UART};
+use caliptra_hw_model::{HwModel, InitParams};
+
+#[test]
+fn smoke_test() {
```

Comment:
Where should the integration test bench with the runtime and fmc crates reside?

Reply:
Agree. This is a very useful outcome.

Reply:
As there's still a lot of unimplemented stuff here, I've decided to remove this commit from the PR. Will follow up in a future PR.